src/js/common/utils/sticky-action-box.js

```diff
@@ -0,0 +1,14 @@
+window.onload = function stickyActionBox() {
+  const actionBox = document.querySelectorAll('.show-for-large-up .sticky-action-box')[0];
+  const offset = actionBox.getBoundingClientRect();
+
+  window.addEventListener('scroll', () => {
```

Comment:
Good point, I was worried about this being called too many times too. I'll add that. And look into it that bug!